src/SDK/MonitoringApplianceEmulator/Controls/TracesControl.xaml

```diff
@@ -56,11 +56,13 @@
                             <Setter Property="Foreground" Value="DarkGray"/>
                         </DataTrigger>
                         <DataTrigger Binding="{Binding Level}" Value="Warning">
-                            <Setter Property="Background" Value="Orange"/>
+                            <!--<Setter Property="Background" Value="Orange"/>-->
+                            <Setter Property="Foreground" Value="DarkOrange"/>
                         </DataTrigger>
                         <DataTrigger Binding="{Binding Level}" Value="Error">
-                            <Setter Property="Background" Value="Red"/>
-                            <Setter Property="Foreground" Value="White"/>
+                            <!--<Setter Property="Background" Value="Red"/>
```

Comment:
><!--<Setter Property="Background" Value="Red"/> [](start = 28, length = 47)

why keeping these comments?

Reply:
oops :)

---
In reply to: [232486549](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/20#discussion_r232486549) [](ancestors = 232486549)